src/components/DocumentConfigurationTab.tsx

```
@@ -0,0 +1,351 @@
+					setDocumentConfigs([
+						{
+							id: Date.now(),
+							name: '',
+							label: '',
+							documentSubType: '',
+							docType: '',
+							vcFields: '',
+						},
+					]);
+				}
+			} catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)